File: src/main/java/eterea/api/rest/model/dto/ProgramaDiaDTO.java
/**
 * 
 */
package eterea.api.rest.model.dto;

import java.io.Serializable;
import java.util.List;

import eterea.api.rest.model.ClienteMovimiento;
import eterea.api.rest.model.ReservaOrigen;
import eterea.api.rest.model.Voucher;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class ProgramaDiaDTO implements Serializable {

	private static final long serialVersionUID = 1559102796812991435L;
	
	private List<Voucher> vouchers;
	private List<ReservaOrigen> reservaOrigens;
	private List<ClienteMovimiento> clienteMovimientos;

}

File: src/main/java/eterea/core/api/rest/kotlin/model/dto/ProgramaDiaDTO.kt
package eterea.core.api.rest.kotlin.model.dto

import eterea.core.api.rest.kotlin.model.ClienteMovimiento
import eterea.core.api.rest.kotlin.model.ReservaOrigen
import eterea.core.api.rest.kotlin.model.Voucher

data class ProgramaDiaDTO(

    var vouchers: List<Voucher?>? = null,
    var reservaOrigens: List<ReservaOrigen?>? = null,
    var clienteMovimientos: List<ClienteMovimiento?>? = null

)
